src/test/java/com/glovoapp/planningpoker/e2e/pages/SessionPage.java

```diff
@@ -0,0 +1,233 @@
+package com.glovoapp.planningpoker.e2e.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.awt.*;
+import java.awt.datatransfer.DataFlavor;
+import java.awt.datatransfer.UnsupportedFlavorException;
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.BinaryOperator;
+import java.util.function.Predicate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfElementLocated;
+
+public final class SessionPage {
+
+    private final WebDriver driver;
+
+    public SessionPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    private static final By YOUR_NAME_SELECTOR = By.id("my-name");
+    private static final By SPECTATOR_SELECTOR = By.id("is-spectator");
+    private static final By TICKET_SELECTOR = By.id("ticket-name");
+    private static final By ERROR_MESSAGE_SELECTOR = By.id("error-message");
+    private static final By AUTO_RELOAD_MESSAGE_SELECTOR = By.id("auto-reload-message");
+
+    private static final By CLEAR_BUTTON_SELECTOR = By.id("clear-all");
+    private static final By SHOW_VOTES_SELECTOR = By.id("show-votes");
+    private static final By SHARE_SESSION_LINK_SELECTOR = By.id("copy-session-link-to-clipboard");
+
+    private static final By YOUR_VOTE_SELECTOR = By.id("current-vote");
+    private static final By VOTE_BUTTONS_SELECTOR = By.id("vote-buttons");
+    private static final By MEDIAN_VOTES_SELECTOR = By.id("median-votes-text");
+    private static final By AVERAGE_VOTES_SELECTOR = By.id("average-votes-text");
+    private static final By VOTES_TABLE_SELECTOR = By.id("votes-table");
+    private static final By DANCE_SELECTOR = By.id("glopi-dance");
+
+    public void waitForError() {
+        final WebDriverWait wait = new WebDriverWait(driver, 2);
+        wait.until(visibilityOfElementLocated(ERROR_MESSAGE_SELECTOR));
+    }
+
+    private WebElement danceSection() {
+        return driver.findElement(DANCE_SELECTOR);
+    }
+
+    public void isHeReallyDancing() {
+        danceSection().findElement(By.tagName("img")).isDisplayed();
```

Comment:
This method doesn't do anything. Why not perform an assert?

```suggestion
        assertTrue(danceSection().findElement(By.tagName("img")).isDisplayed());
```